requirements-dev.lock

```diff
@@ -18,6 +18,10 @@ coverage==7.4.4
     # via pytest-cov
 emoji==2.10.1
     # via signal-export
+exceptiongroup==1.2.2
+    # via pytest
+importlib-metadata==8.5.0
```

Comment:
Please remove the changes to both requirements files. importlib-metadata isn't needed, since we don't support Python < 3.8.

I'm not sure why exceptiongroup/tomli/zipp were added, but if there's something necessary there it can come in another PR please.